src/Slices/ServiceInventory/UI/Components/RowActionsMenu/SetStateSection/SetStateSection.tsx

```diff
@@ -53,6 +50,32 @@ export const SetStateSection: React.FunctionComponent<Props> = ({
     if (Maybe.isSome(result)) {
       setStateErrorMessage(result.value);
     }
+    closeModal();
+  };
+
+  const openModal = () => {
+    triggerModal({
+      title: words("inventory.statustab.confirmTitle"),
+      actions: [
+        <Button
+          key="confirm"
+          variant="primary"
+          data-testid={`${id}-state-modal-confirm`}
+          onClick={() => onSubmit(targetState)}
+        >
+          {words("yes")}
+        </Button>,
+        <Button
+          key="cancel"
+          variant="link"
```

Comment:
link? not secondary?

Reply:
I follow what it was already, and modals that don't use confirm action form had link variant